book/src/installers/customizing.md

```diff
@@ -0,0 +1,109 @@
+# Customizing
+
+> since 0.12.0
+
+axo's installers are designed to require no end user input- however, in many
+cases, end users like the ability to tailor their installation experience. For
+these users, our installers allow several inputs which give the user more
+control over their installation experience via configuration. Most of these are
+implemented using environment variables, however several are also available as
+arguments to the installer scripts themselves.
+
+Several of these options were available in earlier releases of `dist`, but they
```

Comment:
Do we want to call ourselves `cargo-dist` now, or start letting new docs drift towards `dist`?

Reply:
i've been drifting towards dist to save the labor for the upcoming rename :)